oauth2l/__init__.py

```diff
@@ -256,6 +263,23 @@ def _GetCredentialsVia3LO(client_info, credentials_filename=None):
             credentials = None
     return credentials
 
+def _GetCredentialForServiceAccount(json_keyfile, scopes, 
```

Comment:
Add GetCredentialStore to merge these two piece of code. The idea is the same as your LookupCache example.